src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -117,7 +120,17 @@ inline auto _make_storage(const std::string &path) {
           sqlite_orm::make_column("id", &DBAccessKey::id, sqlite_orm::autoincrement(), sqlite_orm::primary_key()),
           sqlite_orm::make_column("access_key", &DBAccessKey::access_key),
           sqlite_orm::make_column("user_id", &DBAccessKey::user_id),
-          sqlite_orm::foreign_key(&DBAccessKey::user_id).references(&DBUser::user_id))
+          sqlite_orm::foreign_key(&DBAccessKey::user_id).references(&DBUser::user_id)),
+    sqlite_orm::make_table(std::string(LC_HEAD_TABLE),
+          sqlite_orm::make_column("lc_index", &DBOPLCHead::lc_index, sqlite_orm::primary_key()),
+          sqlite_orm::make_column("marker", &DBOPLCHead::marker),
+          sqlite_orm::make_column("start_date", &DBOPLCHead::start_date)),
+    sqlite_orm::make_table(std::string(LC_ENTRIES_TABLE),
+          sqlite_orm::make_column("lc_index", &DBOPLEntry::lc_index),
```

Comment:
any particular reason why we have a `DBOPLCHead` (note the `LC`), but here we have a `DBOPLEntry` (without the `C` in `LC`)?

Reply:
nah, just the classic typo that spreads like a virus after glorious copy-paste... I'll fix it